src/test/java/org/broadinstitute/hellbender/utils/pileup/ReadPileupUnitTest.java

```diff
@@ -490,4 +491,106 @@ public void testSimplePileupWithOffset(){
         Assert.assertTrue(pu.makeFilteredPileup(p -> p.getMappingQual() >= 10).isEmpty());
     }
 
+    @DataProvider(name = "FixPairOverlappingQualitiesTest")
+    public Object[][] overlappingElementsToFix() throws Exception {
+        final byte highQual = 60;
+        final byte lowQual = 10;
+        final byte qualitySum = 70;
+        final byte reducedQuality = 48; // 80% of 60
+        final byte zeroQuality = 0;
+        final GATKRead read1 = ArtificialReadUtils.createArtificialRead("5M");
+        final GATKRead read2 = ArtificialReadUtils.createArtificialRead("5M");
+        read1.setBases(new byte[] {'A', 'A', 'A', 'A', 'T'});
+        read1.setBaseQualities(new byte[] {highQual, highQual, highQual, lowQual,
+                SAMUtils.MAX_PHRED_SCORE});
+        read2.setBases(new byte[] {'A', 'T', 'T', 'T', 'T'});
+        read2.setBaseQualities(new byte[] {lowQual, highQual, lowQual, highQual,
+                SAMUtils.MAX_PHRED_SCORE});
+        return new Object[][] {
+                {PileupElement.createPileupForReadAndOffset(read1, 0),
```

Comment:
For each of these tests, could you provide a 1 sentence description of what that test is establishing?